src/fmacros.h

```diff
@@ -52,7 +52,7 @@
 #define _XOPEN_SOURCE 700
 /*
  * On NetBSD, _XOPEN_SOURCE undefines _NETBSD_SOURCE and
- * thus hides inet_aton etc.
+ * thus hides inet_aton, etc.
```

Comment:
I have half a rule for `, etc.` but I didn't actually deploy it for this repository. This ended up being an in-the-neighborhood correction (I was checking up on a heuristic to tolerate `aton` and spotted it).